mappings/net/minecraft/fluid/BaseFluid.mapping

```diff
@@ -2,11 +2,12 @@ CLASS net/minecraft/class_3609 net/minecraft/fluid/BaseFluid
 	FIELD field_15900 LEVEL Lnet/minecraft/class_2758;
 	FIELD field_15902 FALLING Lnet/minecraft/class_2746;
 	FIELD field_17587 shapeCache Ljava/util/Map;
-	METHOD method_15725 (Lnet/minecraft/class_1936;Lnet/minecraft/class_2338;Lnet/minecraft/class_3610;)V
+	METHOD method_15725 tryFlow (Lnet/minecraft/class_1936;Lnet/minecraft/class_2338;Lnet/minecraft/class_3610;)V
 		ARG 1 world
-		ARG 2 pos
+		ARG 2 fluidPos
```

Comment:
🤔 what does this distinguish?

Reply:
This has 8 parameters , 2 of which are blockpos